src/components/mc/cpu/mc_cpu.c

```diff
@@ -9,35 +9,153 @@
 #include "utils/ucc_malloc.h"
 #include <sys/types.h>
 
+static ucc_status_t ucc_mc_cpu_mem_alloc_pool_with_init(ucc_mc_buffer_header_t **ptr,
+                                                   size_t size);
+
 static ucc_config_field_t ucc_mc_cpu_config_table[] = {
     {"", "", NULL, ucc_offsetof(ucc_mc_cpu_config_t, super),
      UCC_CONFIG_TYPE_TABLE(ucc_mc_config_table)},
 
-    {NULL}};
+    {"ELEM_SIZE", "1048576", "The size of each element in mc cpu mpool",
```

Comment:
@shimmybalsam @Sergei-Lebedev i would even call it MPOOL_ELEM_SIZE, MPOOL_MAX_ELEMS because this is what it is )) also it will explicitly mean it is pre-allocated pool.